i18n/en.json

```diff
@@ -1547,13 +1546,12 @@
   "privacy": "Privacy",
   "profile": "Profile",
   "profile_drawer_app_logs": "Logs",
-  "profile_drawer_client_out_of_date_major": "Mobile App is out of date. Please update to the latest major version.",
-  "profile_drawer_client_out_of_date_minor": "Mobile App is out of date. Please update to the latest minor version.",
+  "profile_drawer_client_out_of_date": "An app update is available.",
   "profile_drawer_client_server_up_to_date": "Client and Server are up-to-date",
   "profile_drawer_github": "GitHub",
   "profile_drawer_readonly_mode": "Read-only mode enabled. Long-press the user avatar icon to exit.",
-  "profile_drawer_server_out_of_date_major": "Server is out of date. Please update to the latest major version.",
-  "profile_drawer_server_out_of_date_minor": "Server is out of date. Please update to the latest minor version.",
+  "profile_drawer_server_out_of_date": "A server update is available.",
+  "profile_drawer_unable_to_check_version": "Unable to check app or server version",
```

Comment:
I wonder if we should have a way to mark this as needing to be translated again, since it'll keep being the old value for non-en languages.

Reply:
I think just removing these old keys and using new keys with simplified key names like `out_of_date_message`...etc